edg/parts/Camera_Ov2640_Fpc24.py

```diff
@@ -11,11 +11,13 @@ def __init__(self) -> None:
         self.dgnd = self.Export(self.conn.pins.request('10').adapt_to(Ground()))
         self.agnd = self.Export(self.conn.pins.request('23').adapt_to(Ground()))
         self.dovdd = self.Export(self.conn.pins.request('14').adapt_to(VoltageSink(
-            voltage_limits=(1.71, 3.3)*Volt,  # Table 6, absolute maximum (Table 5) is 4.5v
+            voltage_limits=self.nonstrict_3v3_compatible.then_else(
+                (1.71, 4.5)*Volt,  # Table 6, absolute maximum (Table 5) is 4.5v
+                (1.71, 3.3)*Volt,),
```

Comment:
```suggestion
                (1.71, 3.3)*Volt),
```
stray trailing comma